being-an-adie/learning-plans.md

```diff
@@ -0,0 +1,23 @@
+# Learning Plans
+
+[A Learning Plan](https://drive.google.com/file/d/1I1ljWhcF1RL0ih7La2P26iXhq1QABmFx/view) (LP) is a structure we use to help individualize the academic experience at Ada when a student would benefit from more or different support around particular curriculum content. The objective of an LP is to provide concrete and personalized solutions when you need academic support for any reason. An LP typically lasts 1-3 weeks and could resolve with no further action, or we may suggest extending/reworking the plan. 
+
+## Here are some reasons we might suggest a learning plan, or why you might request one:
+
+- You feel like you’re falling behind the curriculum and need help with pacing and prioritizing.
+- There’s a particular section of the curriculum that isn’t coming naturally or you need more help understanding. 
+- Some big things have been happening in your personal life and your ability to fully focus on Ada isn’t as high right now. 
+- You got a red on your last assignment and we want to make sure you’re moving forward alright with that content
```

Comment:
Nitpick: Either have periods at the end of all of these bullet points with one sentence, or don't!

```suggestion
- You got a red on your last assignment and we want to make sure you’re moving forward alright with that content.
```